### PR TITLE
Optimal Edit Paths Return Section Improved

### LINKED ISSUE #7353: Missing documentation for output of ```optimal_edit_paths``` function
### Current Behavior

The current description in the documentation for the ```optimal_edit_paths``` function is:

```
    edit_paths: list of tuples (node_edit_path, edge_edit_path)
         node_edit_path : list of tuples (u, v) edge_edit_path : list of tuples ((u1, v1), (u2, v2))
    cost : numeric
         Optimal edit path cost (graph edit distance).
```
while it states the type of output, there is no explanation on how to interpret the result.
On stackoverflow there is a [question](https://stackoverflow.com/questions/73696000/how-to-interpret-the-output-of-networkx-optimal-edit-paths) where users don't know how to interpret the output of such function.

### PR DESCRIPTION
**Pull request by vanshika230**
Closes #7353.
I have improved the Returns section of the optimal_edit_paths function based on the discussion on issue #7353. I have also referred to the stackoverflow discussion [here](https://stackoverflow.com/questions/73696000/how-to-interpret-the-output-of-networkx-optimal-edit-paths). Do let me know any improvement. Thankyou!

### CODE REVIEW

File: networkx/algorithms/similarity.py [hunk 1]
```diff
@@ -320,7 +320,11 @@ def optimal_edit_paths(
     -------
     edit_paths : list of tuples (node_edit_path, edge_edit_path)
         node_edit_path : list of tuples (u, v)
+            Indicates node transformations between G1 and G2,
+            u=None or v=None for insertion/deletion.
```
Comment (dschult) [inline networkx/algorithms/similarity.py:324]:
```suggestion
            u is None for insertion.  v is None for deletion.
```

File: networkx/algorithms/similarity.py [hunk 2]
```diff
@@ -320,7 +320,11 @@ def optimal_edit_paths(
...
         edge_edit_path : list of tuples ((u1, v1), (u2, v2))
+            Indicates edge transformations between G1 and G2,
+            (None,(u2,v2)) or ((u1,v1),None) for insertion/deletion.
```
Comment (dschult) [inline networkx/algorithms/similarity.py:327]:
```suggestion
            (None, (u2,v2)) for insertion. ((u1,v1), None) for deletion.
```

**Review by dschult — COMMENTED** [review]
Thanks for this!
I have a very minor suggestion to split the insertion/deletion sentences into two sentences to make clear which is which. I also switch `=` to `is` because it is more like english (and python code comparison with None is supposed to use 'is'). But feel free to leave that as it is instead if you prefer.

**Review by rossbar — APPROVED** [review]
I took the liberty of updating the formatting to improve the rendered html, but the content itself LGTM. Thanks @vanshika230 !

**Review by dschult — APPROVED** [review]

**Pull request merged**